browser-tests/functional-tests/functional.spec.ts

```diff
@@ -201,12 +201,15 @@ test.describe.parallel("Functional tests", () => {
 
   test.describe.parallel("Check Strategic App VC receipt", () => {
     ["Mam", "Dad"].forEach((journeyType) => {
```

Comment:
```suggestion
    [{
      journeyType: "Mam",
      inProgressSpinnerText: "This may take a few minutes.",
      completedSpinnerText: "You can now continue.",
    }, ...].forEach((journeyType) => {
```

So you can remove logic from within the test.